tests/base/test_context.py

```diff
@@ -42,7 +42,8 @@
 class ContextCustomRcTests(TestCase):
 
     def setUp(self):
-        string = dals("""
+        string = dals(
+            """
```

Comment:
Since this got formatted, can we indent the text block below to align with these open quotes?

Reply:
Can we remove dals and just use """ with correctly-formatted inner contents?

Reply:
@dholth, to be honest, I've always thought the `dals` function was a little silly, so I wouldn't mind doing it, but I believe it's better to just stick with existing conventions 🤷